APIServer/appscale/api_server/server.py

```diff
@@ -17,7 +17,7 @@
 from appscale.common.constants import VAR_DIR
 from appscale.common.constants import ZK_PERSISTENT_RECONNECTS
 
-logger = logging.getLogger('appscale-api-server')
+logger = logging.getLogger(__name__)
```

Comment:
I think the `main` function in this module should set the level on the `appscale.api_server` logger instead of the logger defined here so that all of the child loggers are changed.